ASIM/schemas/ASimUserManagement.yaml

```diff
@@ -0,0 +1,111 @@
+Schema:
```

Comment:
Is this included somewhere?
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/56674bfc-c369-4ae9-b0f0-e7dd1c1cf0bf)

Reply:
Yes